QueAns/views.py

```diff
@@ -0,0 +1,47 @@
+from django.http import HttpResponse
+from .models import Voting,Meetings,Satisfaction
+from datetime import datetime
+
+
+def upvote(request,user_id,question_id):
+    if request.method == "POST":
```

Comment:
What would be (and follow up question should be?) sent to the client in case request is get.

Reply:
POST method is used when the given request would change the server-side state i.e. add/change/remove from DB. GET methods are used to retrieve the remote data i.e. if we use GET method we can validate the query in that case too but won't be able to add the entry to DB if the entry is valid. In case the request is get then we can tell if the given entry is valid or not and we will return the request stating the same.

Reply:
What happens here?